src/libraries/System.Net.Security/src/System/Net/Security/NetEventSource.Security.cs

```diff
@@ -378,5 +414,50 @@ private unsafe void WriteEvent(int eventId, string arg1, int arg2, int arg3, int
                 }
             }
         }
+
+        [NonEvent]
+        private unsafe void WriteEvent(int eventId, string arg1, string arg2, int arg3, int arg4, int arg5)
```

Comment:
We usually put these helpers into the base NetEventSource class.

See examples: https://github.com/dotnet/runtime/blob/master/src/libraries/Common/src/System/Net/Logging/NetEventSource.Common.cs

Reply:
Should I move the other WriteEvent() as well? I saw them in both places...

Reply:
>I saw them in both places...

What are "both" places?

I think all the `WriteEvent` helpers should be in the base class.

Reply:
https://github.com/dotnet/runtime/blob/184f49b20ee9fc6c2e6649e22750cb6c54ffdc53/src/libraries/System.Net.Security/src/System/Net/Security/NetEventSource.Security.cs#L324-L325

There is already WriteEvent() overload in NetEventSource.Security.cs in addition to NetEventSource.Common.cs

Having everything in base class make sense to me.

Reply:
updated.